releases/release-1.22/release-notes/maps/pr-102620-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 102620
+releasenote:
+  text: Disable log sampling when using json logging format.
```

Comment:
Can we mention in which component is this change?

Reply:
The changes happen in component-base which is shared among multiple component. Please let me know if any component name you think should be added.

Reply:
Yep, it will affect all components, you are correct it will get too much if we list all components.